docs/deploy.md

```diff
@@ -94,8 +94,11 @@ Run through them again for a second cluster to use with the extended example sho
       --set=clusterVpcId= \
       --set=clusterName= \
       --set=awsAccountId= \
+      # You need to pass defaultServiceNetwork if you want the default service network feature, you could check environment.md for more its details
```

Comment:
I don't think we need "You need to pass defaultServiceNetwork if you want the default service network feature" part, "Check environment.md for more details" looks sufficient to me